src/Trixi.jl

```diff
@@ -15,13 +15,15 @@ module Trixi
 # Include other packages that are used in Trixi
 # (standard library packages first, other packages next, all of them sorted alphabetically)
 using LinearAlgebra: dot
-using Pkg.TOML: parsefile
+using Pkg.TOML: parsefile, parse
 using Printf: @printf, @sprintf, println
 using Profile: clear_malloc_data
 using Random: seed!
 
 using EllipsisNotation
 using HDF5: h5open, attrs
+using MPI # We use all symbols, but for now we always prefix with `MPI.`, e.g., `MPI.Init()`
```

Comment:
You could also `import MPI` to force adding `MPI.` in front of everything `export`ed from `MPI`.

Reply:
Thanks! Fixed in 40874dd.